### DISCUSSION TITLE
why github dosent include my name as contributor

### DISCUSSION THREAD

**Discussion by Ahmedhenchiri** [community/community#41398] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

why github dosent include my name as contributor

**Comment by mohamedzhioua** [comment, marked as answer]
It’s not GitHub’s or Git’s fault. You should set your name and email address when you commit your changes in your PC.

To do that, run the following commands:

git config user.name “Firstname Lastname”

git config user.email “username@emailprovider.com”
